<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MKBExSubclassDialog</class>
 <widget class="QDialog" name="MKBExSubclassDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расширенная субклассификация МКБ</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Расширенная субклассификация МКБ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CTableView" name="tblExSubclass"/>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Вставить F9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEdit">
         <property name="text">
          <string>Правка F4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>элемент субклассификации</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CTableView" name="tblExSubclass_Item"/>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdd_Item">
         <property name="text">
          <string>Вставить F9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEdit_Item">
         <property name="text">
          <string>Правка F4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDel_Item">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
